src/core/client/stream/tabs/Comments/Stream/CommentForm/CommentForm.css

```diff
@@ -3,6 +3,7 @@ $commentsBorder: var(--palette-grey-300);
 .commentFormBox {
   border: 1px solid $commentsBorder;
   border-radius: var(--round-corners);
+  overflow: hidden;
```

Comment:
this fixes a tiny bug where the rounded corners were being clipped by a child with a background